docs/.gitignore

```diff
@@ -0,0 +1,4 @@
+*
+!.gitignore
```

Comment:
Not sure why you need to ignore everything under this folder by default.

I prefer the normal way to explicitely specify what you ignore to avoid ignoring "too much" and forgot to commit new files.

Reply:
I'm not sure if this is a remnant of an older commit build of the docs, but I had a lot of tmp doc files all over that directory (as in dozens of files). Running `make clean` removes them, but there is still a chance one could do `git add docs` and have lots of garbage collected. I find it is safe to do this kind of include as it is expected all doc files should be put in `sources` directory.

Reply:
That's odd, what make command generate those cruft initially?  Maybe should fix the source of the problem instead (make all the temporary output under some dir and only add that specific dir to `.gitignore`.

Oh, I found the answer to your problem: https://github.com/bird-house/birdhouse-deploy/blob/57a320c3583f804838e8c18aeb064266527a0bc9/docs/Makefile#L50-L51

https://github.com/bird-house/birdhouse-deploy/blob/57a320c3583f804838e8c18aeb064266527a0bc9/docs/Makefile#L8

So you should just add folder `_build` to the `.gitignore` instead of ignoring everything.

Reply:
I can't tell. I'm also expecting all of them to be in `_build` since this is what is defined in Makefile, but I did have lots of files there. Like I said, maybe its due to old run. Either way, I don't have an objection, I just used that because I didn't want to include all the files in .gitignore manually.

Reply:
But you said `make clean` removes them so all the files must have been under `_build`.  If files outside of `_build` meaning they are not created by this repo so why should `.gitignore` of this repo take care of them?

So basically if a `.gitignore` is required, I think only `_build` should be in there.  A "ignore everything" rule is dangerous and in this case not appropriate since the repo should only handle files it creates itself.

Reply:
I don't feel the dangerous aspect of it since there shouldn't be anything there anyway. All docs should be placed in `source`. Anyway, I will apply more specific `_build` if you prefer that. Same result for me.